### ISSUE TITLE
Also delete this one

### ISSUE THREAD

**Issue by jgraham** [w3c/webdriver#832]
In chapter [8. Sessions](https://w3c.github.io/webdriver/webdriver-spec.html#new-session#h-sessions), section [8.1 New Session](https://w3c.github.io/webdriver/webdriver-spec.html#new-session#h-new-session):
> Let capabilities be capabilities result’s data. 

Sorry should have read further before starting to file issues.

**Comment by AutomatedTester** [comment]
duplicate of #830
